acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -101,5 +115,42 @@ protected AcceptanceTestBase() {
   @After
   public void tearDownAcceptanceTestBase() {
     cluster.close();
+
+    String os = System.getProperty("os.name");
```

Comment:
Apply the clean code philosophy here, refactor these into private methods with descriptive names, please

Reply:
Done.

Reply:
>Done.

My observation is that the code has yet to change 🤔

Reply:
This is weird. In the "Conversation" tab of GitHub, this thread shows the code as it was originally, on Feb 20. It ignores the changes I introduced Feb 23 (commit 497c15b)
 
Meanwhile, in the "Files changed" tab, this thread does show those changes.

I wonder whether we were looking at the code from different tabs then? Or did you really mean that you wanted even more cleaning?

Anyway, I have just pushed some more cleaning/refactoring that I did before realizing the GitHub weirdness.

Reply:
> Or did you really mean that you wanted even more cleaning?

That one 😁

Reply:
so... done now?